lib/phoenix_live_view/test/client_proxy.ex

```diff
@@ -969,8 +969,10 @@ defmodule Phoenix.LiveViewTest.ClientProxy do
     end
   end
 
-  defp maybe_event(:click, {"a", _, _} = node, element) do
-    live_nav = DOM.attribute(node, "data-phx-link")
+  defp maybe_event(:click, {tag, _, _} = node, element) do
+    live_nav =
+      DOM.attribute(node, "data-phx-link") ||
```

Comment:
In the browser, wouldn't phx-click win over data-phx-link?

Reply:
Just checking about which one should win here. I think phx-click but I am not sure.